models/posts.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+let posts = new mongoose.Schema({
+  title: String,
+  desc: String,
+  comments: [{ type: mongoose.Schema.Types.ObjectId, ref: "comment" }],
+  userid: { type: mongoose.Schema.Types.ObjectId, ref: "user" },
```

Comment:
In comment model you've used `user` and here you are using `userid`. Please be consistent on naming.
`user` will be fine here.

Reply:
oksy